dataloader/dataloader.hpp

```diff
@@ -0,0 +1,225 @@
+/**
+ * @file dataloader.hpp
+ * @author Kartik Dutt
+ * 
+ * Definition of Dataloader for popular datasets.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core/data/scaler_methods/min_max_scaler.hpp>
+#include <mlpack/core/math/shuffle_data.hpp>
+#include <mlpack/core/data/split_data.hpp>
+#include <mlpack/prereqs.hpp>
+#include <dataloader/datasets.hpp>
+#include <mlpack/core.hpp>
+#include <utils/utils.hpp>
+
+
+#ifndef MODELS_DATALOADER_HPP
+#define MODELS_DATALOADER_HPP
+
+/**
+ * Dataloader class to load popular datasets.
+ *
```

Comment:
Maybe a simple example here would be nice to have, similar to:

https://github.com/mlpack/mlpack/blob/9672d5d15368aca5600e6f72cb8f2fd15d2bc25d/src/mlpack/methods/mean_shift/mean_shift.hpp#L32-L41

Reply:
Yes, will add it in the next commit. Thanks a lot for the review.